diff -urNp -x '*.orig' Sphinx-1.8.5.org/setup.py Sphinx-1.8.5/setup.py
--- Sphinx-1.8.5.org/setup.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/setup.py	2021-02-27 11:42:43.620291111 +0100
@@ -43,7 +43,6 @@ extras_require = {
         'whoosh>=2.0',
     ],
     'test': [
-        'mock',
         'pytest',
         'pytest-cov',
         'html5lib',
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_build.py Sphinx-1.8.5/tests/test_build.py
--- Sphinx-1.8.5.org/tests/test_build.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_build.py	2021-02-27 11:42:43.620291111 +0100
@@ -13,7 +13,10 @@ import pickle
 import sys
 from textwrap import dedent
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 from docutils import nodes
 
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_config.py Sphinx-1.8.5/tests/test_config.py
--- Sphinx-1.8.5.org/tests/test_config.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_config.py	2021-02-27 11:42:43.620291111 +0100
@@ -9,7 +9,10 @@
     :copyright: Copyright 2007-2019 by the Sphinx team, see AUTHORS.
     :license: BSD, see LICENSE for details.
 """
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 from six import PY3
 
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_domain_std.py Sphinx-1.8.5/tests/test_domain_std.py
--- Sphinx-1.8.5.org/tests/test_domain_std.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_domain_std.py	2021-02-27 11:42:43.620291111 +0100
@@ -9,7 +9,10 @@
     :license: BSD, see LICENSE for details.
 """
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 from docutils import nodes
 
 from sphinx.domains.std import StandardDomain
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_environment_indexentries.py Sphinx-1.8.5/tests/test_environment_indexentries.py
--- Sphinx-1.8.5.org/tests/test_environment_indexentries.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_environment_indexentries.py	2021-02-27 11:42:43.620291111 +0100
@@ -11,7 +11,10 @@
 
 from collections import namedtuple
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from sphinx import locale
 from sphinx.environment.adapters.indexentries import IndexEntries
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_ext_intersphinx.py Sphinx-1.8.5/tests/test_ext_intersphinx.py
--- Sphinx-1.8.5.org/tests/test_ext_intersphinx.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_ext_intersphinx.py	2021-02-27 11:42:43.620291111 +0100
@@ -13,7 +13,10 @@ import os
 import unittest
 from io import BytesIO
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import requests
 from docutils import nodes
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_ext_napoleon.py Sphinx-1.8.5/tests/test_ext_napoleon.py
--- Sphinx-1.8.5.org/tests/test_ext_napoleon.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_ext_napoleon.py	2021-02-27 11:42:43.620291111 +0100
@@ -13,7 +13,10 @@
 from collections import namedtuple
 from unittest import TestCase
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from sphinx.application import Sphinx
 from sphinx.ext.napoleon import _process_docstring, _skip_member, Config, setup
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_ext_napoleon_docstring.py Sphinx-1.8.5/tests/test_ext_napoleon_docstring.py
--- Sphinx-1.8.5.org/tests/test_ext_napoleon_docstring.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_ext_napoleon_docstring.py	2021-02-27 11:42:43.620291111 +0100
@@ -15,7 +15,10 @@ from inspect import cleandoc
 from textwrap import dedent
 from unittest import TestCase
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from sphinx.ext.napoleon import Config
 from sphinx.ext.napoleon.docstring import GoogleDocstring, NumpyDocstring
diff -urNp -x '*.orig' Sphinx-1.8.5.org/tests/test_util_fileutil.py Sphinx-1.8.5/tests/test_util_fileutil.py
--- Sphinx-1.8.5.org/tests/test_util_fileutil.py	2019-03-10 08:47:02.000000000 +0100
+++ Sphinx-1.8.5/tests/test_util_fileutil.py	2021-02-27 11:42:43.620291111 +0100
@@ -9,7 +9,10 @@
     :license: BSD, see LICENSE for details.
 """
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from sphinx.jinja2glue import BuiltinTemplateLoader
 from sphinx.util.fileutil import copy_asset, copy_asset_file
